core/src/saros/editor/IEditorManager.java

```diff
@@ -53,6 +54,19 @@
    */
   String getContent(SPath path);
 
+  /**
+   * Returns the normalized text content of the local editor associated with the specified file, or
```

Comment:
Maybe I am reading it wrong, but when I have to implement this I would just return the non normalized content if the file is not open in any editor and so this would lead to inconsistencies.

Looking at the return description it states that the file content has to be normalized as well.

Reply:
I just copied it from the other description. But yes, the description (for both) does not match the functionality. It shoud state that it returns the (normalized) editor content or the file content if the file is not openable in a text editor. That is the current logic if I remember correctly (will check again). Will adjust the javadoc for both.

Reply:
I should have actually looked at the logic before hastily writing a reply from memory. The current logic for Eclipse and IntelliJ **only** returns the editor/document content (even if there is no editor for the resource open). If no such editor/document content can be obtained, `null` is returned. (And the server always returns the "editor" content as it works directly on a text buffer internally.)

Will correct the javadoc.

Reply:
Done. Adjusted the javadoc in additional commit. The new wording is a bit clunky and I am not completely happy with it, but I couldn't come up with anything better. Its hard to properly convey the "document" concept (i.e. the content that would be displayed in a text editor if a text editor were open for the file).